docs/install.rst

```diff
@@ -1,5 +1,6 @@
 Installation
 ============
+``frank`` requires Python >= 3.0 and does not support compatibility with Python 2.x.
```

Comment:
We cannot say that requires Python >=3.0 because it would imply that it works in all Python versions later than 3.0, which span a lot of years back and I am sure it is not the case.
So, I suggest a more specific:
```suggestion
``frank`` requires Python 3 and is tested for Python 3.7. Usage under Python 2.x is not supported.
```
I've opened #116 to address this and discover whether we can support Python 3.6 and 3.8 as well.
I've opened #115 for the analogous with packages.

Reply:
Thanks Marco for catching this and for opening the other issues, definitely important to check these things.